aiidalab/utils.py

```diff
@@ -253,8 +254,17 @@ def run_reentry_scan() -> Any:
 
 
 def run_verdi_daemon_restart() -> Any:
+    # When installing or updating a plugin package, one needs to
+    # restart the daemon with the ``--reset`` flag for changes to take effect.
+    # however, in the latest aiida-core branch, this is now the default.
+    # Define command based on aiida-core version
+    aiida_version = version("aiida-core")
+    if aiida_version <= "2.5.1":
```

Comment:
Why do you need this distinction? I believe the --reset flag is still present in upcoming version no? Or am I wrong?

Reply:
I am not sure about this. This at least covers all our docker images and the latest stable AiiDA. We may update it when the new AiiDA version is available. I can open an issue for reminder.

Do you have any better suggestions to handle it?